NuGetPackageVerifier.json

```diff
@@ -4,16 +4,54 @@
+          },
+          "VERSION_INFORMATIONALVERSION": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg."
+          },
+          "WRONG_PUBLICKEYTOKEN": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "PACKAGE_TYPE_UNEXPECTED": {
+            "*": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_INFORMATIONAL_VERSION_MISMATCH": {
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_FILE_VERSION_MISMATCH": {
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "ASSEMBLY_VERSION_MISMATCH": {
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+          },
+          "FILE_MISSING_FROM_SIGN_REQUEST": {
+            "tools/any/any/Microsoft.Extensions.FileProviders.Abstractions.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/Microsoft.Extensions.FileProviders.Physical.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/Microsoft.Extensions.FileSystemGlobbing.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/Microsoft.Extensions.Primitives.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
+            "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg.",
+            "tools/any/any/toolassets/net461/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll": "Assembly built by another project but bundled in our nupkg.",
```

Comment:
@natemcmaster: this doesn't seem right. This dll is built by the scaffolding repo. Its not packaged as a nupkg, Is there a way/ need to get this signed?

Reply:
Yes. Everything in the package should be signed. 

Unlike DotNetCliTool packages, a DotnetTool package is a "fat package" i.e. anything in the output of dotnet-publish ends up in the nuget package. This means we need to make sure we sign all of our assemblies, and skip assemblies signed by corefx, and 3rd party sign assemblies like Newtonsoft.Json.

Reply:
So, how do I add this?

Reply:
Add these to the .csproj file. Example: https://github.com/aspnet/DotNetTools/blob/d9d4b8042eda0abed5acbdbfdefee7446d7ba3e8/src/dotnet-sql-cache/dotnet-sql-cache.csproj#L15-L22